cmd/launcher/launcher.go

```diff
@@ -92,6 +93,11 @@ func main() {
 		logFatal(logger, "err", errors.Wrap(err, "detecting platform"))
 	}
 
+	debugTokenPath := filepath.Join(rootDirectory, "debug_token")
+	if err := debug.StartDebugServer("localhost:5097", debugTokenPath, logger); err != nil {
```

Comment:
the port should not be hardcoded. We're going to end up failing on a bunch of machines just because. It should be a random available port IMO.